package.json

```diff
@@ -37,15 +38,24 @@
         "visible": true
       },
       {
-        "permissionName": "ui-orders.view",
-        "displayName": "Orders: View orders and PO lines",
+        "permissionName": "ui-orders.third-party-services",
```

Comment:
I've decided to move common required in View and Edit permissions in a separate one for reusing